### PR TITLE
Added tlg4072.tlg002.1st1K-grc1.xml

### PR DESCRIPTION
**Pull request by sonofmun**
Eutocius Commentarius in dimensionem circuli

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |     +132 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +3950 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg4072/__cts__.xml`                          | New      |
| `./data/tlg4072/tlg002/__cts__.xml`                   | New      |
| `./data/tlg4072/tlg002/tlg4072.tlg002.1st1K-grc1.xml` | New      |
| `./data/tlg4072/__cts__.xml`                          | Passing  |
| `./data/tlg4072/tlg002/__cts__.xml`                   | Passing  |
| `./data/tlg4072/tlg002/tlg4072.tlg002.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3634)*

**Comment by AlisonBabeu** [comment]
Hi @sonofmun just one little error the file ID should not be 
```xml
<idno type="filename">tlg4072.tlg001.1st1K-grc1.xml</idno>
```
but should be 
```xml
<idno type="filename">tlg4072.tlg002.1st1K-grc1.xml</idno>
```

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |     +132 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +3950 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg4072/__cts__.xml`                          | New      |
| `./data/tlg4072/tlg002/__cts__.xml`                   | New      |
| `./data/tlg4072/tlg002/tlg4072.tlg002.1st1K-grc1.xml` | New      |
| `./data/tlg4072/__cts__.xml`                          | Passing  |
| `./data/tlg4072/tlg002/__cts__.xml`                   | Passing  |
| `./data/tlg4072/tlg002/tlg4072.tlg002.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3642)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |     +132 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +3950 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg4072/__cts__.xml`                          | New      |
| `./data/tlg4072/tlg002/__cts__.xml`                   | New      |
| `./data/tlg4072/tlg002/tlg4072.tlg002.1st1K-grc1.xml` | New      |
| `./data/tlg4072/__cts__.xml`                          | Passing  |
| `./data/tlg4072/tlg002/__cts__.xml`                   | Passing  |
| `./data/tlg4072/tlg002/tlg4072.tlg002.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3648)*

File: data/tlg4072/tlg002/tlg4072.tlg002.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,857 @@
...
+	</persName>
+  </editor>
+  <author ref="urn:cts:greekLit:tlg4072">Eutocius</author>
+  <imprint>
+   <publisher xml:lang="fre">Les Belles Lettres</publisher>
+	<pubPlace>Paris</pubPlace>
+	<date>1972</date>
+  </imprint>
+  <biblScope unit="volume">4</biblScope>
+ </monogr>
```
Comment (planatheisa) [inline data/tlg4072/tlg002/tlg4072.tlg002.1st1K-grc1.xml]: Scan missing.

File: data/tlg4072/tlg002/tlg4072.tlg002.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,857 @@
...
+<listBibl>
+ <biblStruct>
+ <monogr>
+  <title xml:lang="fre">Archimède</title>
+  <editor>
+	<persName>
+	 <name>Charles Mugler</name>
+	</persName>
+  </editor>
+  <author ref="urn:cts:greekLit:tlg4072">Eutocius</author>
```
Comment (planatheisa) [inline data/tlg4072/tlg002/tlg4072.tlg002.1st1K-grc1.xml]: Should have `xml:lang="lat"`.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |     +132 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +3950 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg4072/__cts__.xml`                          | New      |
| `./data/tlg4072/tlg002/__cts__.xml`                   | New      |
| `./data/tlg4072/tlg002/tlg4072.tlg002.1st1K-grc1.xml` | New      |
| `./data/tlg4072/__cts__.xml`                          | Passing  |
| `./data/tlg4072/tlg002/__cts__.xml`                   | Passing  |
| `./data/tlg4072/tlg002/tlg4072.tlg002.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3668)*

**Pull request merged**